interfaces/builtin/network_control.go

```diff
@@ -160,6 +164,21 @@ capability setuid,
 /bin/run-parts ixr,
 /etc/resolvconf/update.d/* ix,
 
+# wpa_suplicant
+/run/wpa_supplicant/** rw,
+/var/run/wpa_supplicant/** rw,
+/etc/wpa_supplicant/{,**} ixr,
+
+#ifup,ifdown, dhclient
+/run/dhclient.*.pid rw,
```

Comment:
You referenced /var/run for wpa_supplicant, we should do the same here:
```
/{,var/}run/dhclient.*.pid rw,
```

Reply:
changed